test/planning/test_graph_search.py

```diff
@@ -0,0 +1,51 @@
+from pyroboplan.core.utils import configuration_distance
+from pyroboplan.planning.graph import Node, Graph
+from pyroboplan.planning.graph_search import astar, dfs
+
+
+def construct_graph(N):
+    """Creates a connected NxN graph for testing."""
```

Comment:
Would be useful to draw a simple ASCII description of this graph, like 5x5, to explain what this is doing.

Would highly recommend building the docs to see how it renders lol